DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -905,6 +907,12 @@ static function bool BingoGoalImpossible(string bingo_event, int start_map, int
     case 2: // NSF Generator
         switch(bingo_event)
         {
+        case "KnowYourEnemy":
```

Comment:
if we do a 2/2/2 split then we don't need these special cases for "KnowYourEnemy" since the math will work ok for them, but we can maybe keep the rest of this code to be prepared for other similar situations

Reply:
so we don't need this section anymore
```suggestion
```